### ISSUE TITLE
Creation of the final presentation

### ISSUE THREAD

**Issue by taborda11** [sigevo-summer-school-2018/tigers#1]
Creation of the final presentation for **19th July 2018**.

**Comment by JJ** [comment]
Of course, you are right. I didn't get notified of the creation of the
milestone. Sorry about that.

**Comment by kento031** [comment]
Investigation of the related works and backgrounds
